src/views/microworldshomepage/l10n.json

```diff
@@ -0,0 +1,15 @@
+{
+    "ttt.placeholder": "Placeholder text",
```

Comment:
Is this used anywhere? We shouldn't ask the translators to translate temporary text.